fe/fe-core/src/main/java/com/starrocks/connector/ConnectorPartitionTraits.java

```diff
@@ -281,6 +411,55 @@ public String getTableName() {
         PartitionKey createEmptyKey() {
             return new IcebergPartitionKey();
         }
+
+        @Override
+        public Optional<Long> maxPartitionRefreshTs() {
+            IcebergTable icebergTable = (IcebergTable) table;
+            return Optional.of(icebergTable.getRefreshSnapshotTime());
+        }
+
+        @Override
+        public Set<String> getUpdatedPartitionNames(MaterializedView mv) {
```

Comment:
MV contains a lot of information, is it necessary to make the entire MV as the argument?

Reply:
fixed